configuration/synthesis.tcl

```diff
@@ -33,3 +33,7 @@ set ::env(IO_PCT) 0.2
 set ::env(SYNTH_EXTRA_MAPPING_FILE) ""
 
 set ::env(BASE_SDC_FILE) $::env(SCRIPTS_DIR)/base.sdc
+
+set ::env(QUIT_ON_VERILATOR_WARNINGS) 0
+set ::env(QUIT_ON_VERILATOR_ERRORS) 1
+set ::env(VERILATOR_VERIFIED_PDKS) "sky130A sky130B"
```

Comment:
I don't think this should be a variable, rather, this should be built into the step itself.